### ISSUE TITLE
Customizing image split size away from default 4GB

### ISSUE THREAD

**Issue by ttt113326** [rescuezilla/rescuezilla#267]
Sometimes it is convenient to store a disk image archive in a storage that has a size limit for one file.  Consider the choice before saving: choose the size of a single file of a multivolume archive. 600Mb 1000Mb 1800Mb 2000Mb or manually specify the size of one file in megabytes

**Comment by shasheene** [comment]
Yep, the current split size is 4GB (max file size on legacy FAT32 filesystems, which are still widely used in some circumstances).

I have considered adding the ability to customize split size, but haven't implemented yet as it's been low priority. It's great to have this feature captured as a separate GitHub task.

This task is somewhat related to #9 (burn images to DVD), as the ability to customize this value may be useful to burn images eg, 700MB (CD), 4.7GB (DVD), 25GB (Blu-Ray disks size?) etc.

**Comment by ttt113326** [comment]
I upload to my favorites Telegram messenger.  in Telegram, unlimited cloud storage, but one file must be no more than 2GB.  In order not to be mistaken, for myself I break it into 1950 MB - so it will definitely be within 2GB
